app/scripts/services/validateRegistrant.js

```diff
@@ -73,6 +82,19 @@ angular
                 ? parseFloat(rule.value)
                 : rule.value;
             }
+            // Hide this block if the question that the rule is based on (parentBlock) is hidden.
+            let parentBlock = _.find(blocks, { id: rule.parentBlockId });
+            if (
+              conference &&
+              !blockVisibleRuleCheck(
+                parentBlock,
+                registrant,
+                ruleType,
+                conference,
+              )
+            ) {
+              return false;
```

Comment:
Returning false makes `_.forEach` stop iterating.

```suggestion
              return;
```